tools/build_release.py

```diff
@@ -21,109 +20,137 @@
 from optparse import OptionParser
 import json
 from shutil import copy
-from past.builtins import basestring
+from past.builtins import str
```

Comment:
I believe we should revert this line.
```suggestion
from past.builtins import basestring
```